docs/iot/guides/things-api-guide.md

```diff
@@ -45,7 +45,7 @@ st_things_deinitialize();
 printf("ST Things Stack deinitialized\n");
```

Comment:
Just question. Where is this file used? Couldn't find it from iot/toc_all.md.

Reply:
This file is not used anymore. When I searched developer.tizen.org, this file was detected. I'll restore it because we don't need to update it.
I will remove all unused files later.